config/detekt.yml

```diff
@@ -106,6 +106,12 @@ complexity:
     thresholdInObjects: 11
     thresholdInEnums: 11
 
+detekt-rules:
```

Comment:
I tested accessing `BuildConfig.Debug` locally, and the output looked like:

```
Ruleset: detekt-rules - 5min debt
   PropertyAccessBanner - [someVar] at /Users/severinrudie/Desktop/moz-code/main-projects/fenix/app/src/main/java/org/mozilla/fenix/settings/about/AboutPageAdapter.kt:34:23
```

I'm mildly concerned that it looks a _lot_ like a built in rule, and doesn't offer a lot of context.  I could see myself going down an internet rabbit hole if I saw this.  What do you think about renaming the ruleset to `moz-detekt-rules` or similar?

EDIT: the next comment down might be an easier fix to this.